voicebox/connection.py

```diff
@@ -22,14 +22,17 @@ class Connection:
     def __init__(
         self,
         client,
-        packet_handlers: list = []
+        packet_handlers: list,
+        encryption_pipeline: list
```

Comment:
Consider using a more descriptive name for the `encryption_pipeline` variable. It could be renamed to `encryption_handlers` or `encryption_providers` to better reflect its purpose. [medium]